### PR TITLE
dependencies upgraded in jnlp

### PR DESCRIPTION
**Pull request by geralddejong**
The jnlp dependencies now match the new ones from sip-core, prompted by httpclient upgrade.

### CODE REVIEW

**Comment by kiivihal** [comment]
This looks fine to me. I think it can be merged.

**Comment by manuelbernhardt** [comment]
I think so too

File: app/controllers/user/SipCreator.scala [hunk 1]
```diff
@@ -28,6 +28,7 @@ object SipCreator extends DelvingController {
         <vendor>Delving</vendor>
         <description kind="one-line">Delving SIP-Creator</description>
         <icon href={ codebase + "sip-creator-logo.png" } kind="default"/>
+        <icon href={ codebase + "sip-creator-logo.png" } kind="splash"/>
```
Comment (kiivihal) [inline app/controllers/user/SipCreator.scala:31]: What is the purpose of this splash attribute.

Comment (manuelbernhardt) [inline app/controllers/user/SipCreator.scala:31]: I wonder too

Comment (geralddejong) [inline app/controllers/user/SipCreator.scala:31]: It's for when webstart is starting up

Comment (kiivihal) [inline app/controllers/user/SipCreator.scala:31]: thanks for clearing that up. It is ready to merge now

**Pull request merged**